build/yamls/flow-visibility/base/kustomize-config.yaml

```diff
@@ -0,0 +1,3 @@
+varReference:
```

Comment:
Could we add a comment at the beginning of this file about why we need to have this `kustomize-config.yaml` file?